### PR TITLE
master-lps-102636_2

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-102636

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+				key_text = "openGraphDescription",
+				locator1 = "TextArea#ANY",
+				value1 = "${customDescription}");
+		}
+
+		PortletEntry.save();
+	}
+
+	macro viewDefaultConfiguration {
+		AssertElementPresent(
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro]: If there is text in the UI please assert that

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+			key_title = "Open Graph",
+			locator1 = "Header#H4_TITLE");
+
+		AssertElementPresent(
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro]: Same here

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+			key_fieldName = "openGraphTitleEnabled",
+			locator1 = "FormFields#CHECKBOX_LABEL");
+
+		AssertElementPresent(
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/macros/OpenGraph.macro]: And here

File: modules/apps/layout/layout-seo-web-test/src/testFunctional/tests/OpenGraph.testcase [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Pages",
+				configurationName = "SEO",
+				configurationScope = "Virtual Instance Scope");
+
+			FormFields.enableCheckbox(fieldName = "Enable");
+
+			SystemSettings.saveConfiguration();
+		}
```
Comment (austinchiang) [inline modules/apps/layout/layout-seo-web-test/src/testFunctional/tests/OpenGraph.testcase]: Isn't this enabled by default?

**Comment by austinchiang** [comment]
Please resend

**Pull request closed without merging**